app/controllers/admin/config/site_settings_controller.rb

```diff
@@ -21,6 +22,8 @@ def index
         SiteSetting.all_settings(
           filter_names: params[:filter_names],
           filter_area: params[:filter_area],
+          filter_plugin: params[:plugin],
```

Comment:
This is weird, we already allow filtering with plugin. For example what does `https://meta.discourse.org/admin/plugins/chat/settings` send to the server ATM? We should update chat's settings config as well using your new component as a test that plugins can use it too

Reply:
<img width="365" alt="Screenshot 2024-12-17 at 3 49 41 PM" src="https://github.com/user-attachments/assets/f94212e5-a510-40b0-9efd-c2ecaa6fd943" />

yeah, plugins are using that `plugin` param